src/librustc_codegen_utils/codegen_backend.rs

```diff
@@ -161,7 +161,8 @@ impl CodegenBackend for MetadataOnlyCodegenBackend {
                 collector::MonoItemCollectionMode::Eager
             ).0.iter()
         );
-        //::rustc::middle::dependency_format::calculate(tcx);
+        // FIXME: Fix this
+        // ::rustc::middle::dependency_format::calculate(tcx);
```

Comment:
Could the fixme explain what needs to be fixed? Why is that necessary and why doesn't it work?

Reply:
It ices